visreg.js

```diff
@@ -0,0 +1,58 @@
+var wdio = require("webdriverio");
```

Comment:
Just this one? I'm super new to js, so I feel out of my depth. :) should all of the variables be constants?